src/site/changelog/mod.rs

```diff
@@ -32,61 +32,65 @@ fn build_prerelease_toggle(releases: Vec<ReleasesApiResponse>) -> Option<Box<div
     }
 }
 
-pub fn build_page(config: &Config, repo: &str) -> Result<String> {
+pub fn fetch_releases(repo: &str) -> Result<Vec<ReleasesApiResponse>> {
     let repo_parsed = Url::parse(repo)?;
     let parts = repo_parsed.path_segments().map(|c| c.collect::<Vec<_>>());
     if let Some(url_parts) = parts {
-        let mut releases_html: Vec<Box<section<String>>> = vec![];
-        let mut releases_nav: Vec<Box<li<String>>> = vec![];
         let url = format!(
             "https://api.github.com/repos/{}/{}/releases",
             url_parts[0], url_parts[1]
         );
 
         let releases = reqwest::blocking::Client::new()
-            .get(&url)
+            .get(url)
             .header(USER_AGENT, "oranda")
```

Comment:
good call, we may also want to add version number :)

Reply:
How can I get that?

Reply:
found it and done